Sources/Sentry/SentryOptions.m

```diff
@@ -163,6 +164,14 @@ - (void)validateOptions:(NSDictionary<NSString *, id> *)options
     if (nil != options[@"sendDefaultPii"]) {
         self.sendDefaultPii = [options[@"sendDefaultPii"] boolValue];
     }
+
+    if (nil != options[@"tracesSampleRate"]) {
```

Comment:
`m`: We miss validating that the sample rate is between 0 and 1 like we do for `sampleRate`.